devops/compat_ci_exclude.sycl-rel-6_2

```diff
@@ -0,0 +1,75 @@
+# Known OK:
+
+# https://github.com/intel/llvm/pull/18059 made property a no-op, so `FileCheck`
+# fails, but that isn't an actual ABI break.
+DiscardEvents/discard_events_accessors.cpp
+DiscardEvents/discard_events_using_assert_ndebug.cpp
+DiscardEvents/discard_events_usm.cpp
+DiscardEvents/invalid_event.cpp
+DiscardEvents/invalid_event_exceptions.cpp
+
+# https://github.com/intel/llvm/pull/18287
+# Throw exception instead of returning garbage
+Basic/info.cpp
+
+# Likely OK, but need author to provide justification, get approval/confirmation
+# from someone:
+
+# https://github.com/intel/llvm/pull/18565
+# This one should have probably be done in multiple PRs, first improving the
+# test's CHECKs and then making an actual functional change.
+#
+# Based on the title, I'd expect to see reduction in number of ur*retain/release
+# calls, but it only moves a few ur*release CHECKs, so I'll let the author
+# update this explanation.
+KernelAndProgram/disable-caching.cpp
+
+# https://github.com/intel/llvm/pull/18253 broke at least some of the pre-built
+# E2E binaries, but do we really need to provide backward compatibility for
+# binaries built with sanitizers?
+Sanitizer
+
+# https://github.com/intel/llvm/pull/19238
+NonUniformGroups/opportunistic_group.cpp
+
+# https://github.com/intel/llvm/pull/17955, experimental extension
+AsyncAlloc/device/async_alloc_from_pool.cpp
+AsyncAlloc/device/async_alloc_zero_init.cpp
+AsyncAlloc/device/ooo_queue_async_alloc_from_pool.cpp
+
+# Need more investigation by the author:
+
+# https://github.com/intel/llvm/pull/18314
```

Comment:
@Alexandr-Konovalov , looks like your https://github.com/intel/llvm/pull/18314 broke backward compatibility here, see
https://github.com/intel/llvm/actions/runs/16882134385 (using nightly-2025-05-08, pass)
https://github.com/intel/llvm/actions/runs/16882361500 (using nightly-2025-05-09, fail)
Please investigate and either fix (might need backporting to sycl-rel-6_3) or create a PR updating this file describing why it wasn't an ABI-break or was allowed/desired/approved.

Reply:
Thank you, an interesting issue! Opened [SYCL] Add copy/move ctors and assignment operators to sycl::detail::optional #19775.